res/css/_common.pcss

```diff
@@ -481,13 +481,43 @@ legend {
     height: unset;
 }
 
-.mx_Dialog_cancelButton {
-    @mixin customisedCancelButton;
-    width: 18px;
-    height: 18px;
+.mx_Dialog_headerWithCancel div.mx_Dialog_cancelButton {
+  width: 30px;
+  height: 30px;
+  transform: rotate(45deg);
+  position: absolute;
+  top: 10px;
+  right: 0; 
+  
+  &.mx_Dialog_cancelButton {
```

Comment:
This outputs `.mx_Dialog_headerWithCancel div.mx_Dialog_cancelButton.mx_Dialog_cancelButton`, which does not work.